com.dropbox.Client.appdata.xml

```diff
@@ -132,6 +132,10 @@
     </screenshot>
   </screenshots>
 
+  <custom>
+    <value key="GnomeSoftware::key-colors">[(255, 255, 255)]</value>
```

Comment:
> Contrast with the foreground text colour in the Adwaita theme (#2e3436) and in its dark variant (#eeeeec). This generally means choosing a colour with a medium level of brightness.

Plain white does not contrast well with text in `#eeeeec`. This will either want two colours, or a neutral grey.

Reply:
I am not sure how the two color thing works, does it depend on the light/dark theme? In the case of a neutral grey checking those in [Palette](https://gitlab.gnome.org/World/design/palette), would `#9a9996` work?

> You may be able to test by editing the cached appstream xml (and gz) in /var/lib/flatpak.

Looking for the Dropbox name inside, I could not find it so I could not test it and today it is no longer featured.

Reply:
I experimented with this today after getting some tips from Milan, one of the Software maintainers, about how to test this more effectively. Unfortunately it's not actually po

Reply:
OK, I can't push to this PR (can you check "allow commits for maintainers" or w hatever the option is called) but this is what I used:

```
<value key="GnomeSoftware::key-colors">[(20, 200, 235)]</value>
```

Good in light theme:

![Capture d’écran de 2022-01-21 10-12-22](https://user-images.githubusercontent.com/86760/150510393-8de5727d-ca70-46a3-b745-89ec9df7e43b.png)

Bad in dark theme:

![Capture d’écran de 2022-01-21 10-12-34](https://user-images.githubusercontent.com/86760/150510391-89a1bbeb-da90-456e-8f05-d9bdeb85eef8.png)

I think these are both better than the status quo tho:

![Capture d’écran de 2022-01-21 10-21-39](https://user-images.githubusercontent.com/86760/150510536-ceece035-d681-4744-86da-5d4cd9ef1b47.png)
![Capture d’écran de 2022-01-21 10-21-26](https://user-images.githubusercontent.com/86760/150510541-d17779fc-9936-4772-8a5a-e7245a26ec57.png)

Reply:
You should be able to push to my branch, I did not change the option and it is enabled by default.

Reply:
You're quite right. Not sure  what I got wrong!